data/test-cases/plugins/gitlab/subscriptions-and-notifications/Todo_list_member_access_request.md

```diff
@@ -0,0 +1,43 @@
+---
+# (Required) Ensure all values are filled up
+name: "Member access request in the todo list after entering the `/gitlab todo` slash command."
+status: Active
+priority: Normal
+folder: Subscriptions-and-notifications
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Create subscription in any desired channel or DM/Gm on MM to any desired public project in a public group on gitlab.
+2. Create a member access request from a different user to the desired public project on gitlab.
```

Comment:
for which the subscription is created.

Reply:
For subscription here I thought that the todo list is connected with subscriptions, as per the discussion with Raghav i have updated the test cases accordingly.